import "CasterNodeConfig.proto";

message CasterNode {
    required CasterNodeConfig config = 1;
}